contracts/external/cw-bounties/src/tests.rs

```diff
@@ -130,27 +143,42 @@ fn bounty_countract() -> Box<dyn Contract<Empty>> {
 pub fn test_create_bounty() {
     let mut test = Test::new();
 
+    let bounty_amount = 100;
     let balance_before = test
         .app
         .wrap()
-        .query_balance(test.owner.clone(), "ujuno")
+        .query_balance(test.owner.clone(), JUNO_DENOM)
         .unwrap();
     // create bounty
     test.create(
-        coin(100, "ujuno"),
+        coin(bounty_amount, JUNO_DENOM),
         "title".to_string(),
         Some("description".to_string()),
-        &[coin(100, "ujuno")],
+        &[coin(bounty_amount, JUNO_DENOM)],
     )
     .unwrap();
+    // - assert bounty
```

Comment:
additional check on `Bounty` state for making sure all is fine - especially checking in tests for update and payout below.